contents/handbook/small-teams/feature-success/objectives.mdx

```diff
@@ -1,32 +1,29 @@
 As always, reliability is the #1 unwritten goal: Making sure feature flags are reliable trumps every other objective.
 
 
-### Objective: Make it easier for engineers to analyze the success of a feature
+### Objective: Get 5 reference customers for surveys
```

Comment:
```suggestion
### Objective: Get 5 reference customers for Surveys
```

Do we capitalize features? @joethreepwood

Reply:
All features should be written in sentence case